src/validation.cpp

```diff
@@ -4773,8 +4777,11 @@ bool DumpMempool(const CTxMemPool& pool, FopenFn mockable_fopen_function, bool s
         if (!RenameOver(gArgs.GetDataDirNet() / "mempool.dat.new", gArgs.GetDataDirNet() / "mempool.dat")) {
             throw std::runtime_error("Rename failed");
         }
-        int64_t last = GetTimeMicros();
-        LogPrintf("Dumped mempool: %gs to copy, %gs to dump\n", (mid-start)*MICRO, (last-mid)*MICRO);
+        auto last = SteadyClock::now();
+
+        auto copy_secs = Ticks<SecondsDouble>(mid - start);
+        auto dump_secs = Ticks<SecondsDouble>(last - mid);
+        LogPrintf("Dumped mempool: %gs to copy, %gs to dump\n", copy_secs, dump_secs);
```

Comment:
nit: Could keep inlined to not evaluate the args if the log is disabled ?

Reply:
Done as of 6612412dfc